cg/models/downsample/downsample_data.py

```diff
@@ -52,7 +52,7 @@ def downsampled_case_name(
         self,
     ) -> str:
         """Return a case name with _downsampled appended."""
-        return f"{self.case_name}_downsampled"
+        return f"{self.case_name}-downsampled"
```

Comment:
is this needed?

Reply:
I thikn this is what the discussion was about in slack if I remember correctly? @karlnyr wanted this not to be allowed?

Reply:
I think @fevac means if the "-downsampled" affix is needed. I think that it is, because it is the only thing, except for the connected sample, that tells us if this is a real case or not.